containers/unit_tests/TestUnorderedMap.hpp

```diff
@@ -480,6 +480,48 @@ TEST(TEST_CATEGORY, UnorderedMap_consistent_size) {
   ASSERT_EQ(2u, m.size());
 }
 
+namespace Impl {
+
+using map_type = Kokkos::UnorderedMap<int, void, Kokkos::DefaultExecutionSpace>;
+
+KOKKOS_FUNCTION
+void test_insert_to_map_copy(map_type const &input_map, const int i) {
```

Comment:
Does the insertion matter at all or is it just about calling the copy constructor on the device side?